functions/src/on-create-bet.ts

```diff
@@ -58,6 +58,12 @@ export const onCreateBet = functions.firestore
     const bettor = await getUser(bet.userId)
     if (!bettor) return
 
+    await change.ref.update({
+      userAvatarUrl: bettor.avatarUrl,
+      userName: bettor.name,
+      userUsername: bettor.username,
+    })
```

Comment:
Right now, when we create for example a comment, we send some of the denormalized fields in the Firestore create request, like the denormalized user data. But I think it would be better to change those places so that they just sent the normalized version of the object, and Firestore triggers did the denormalization.